test/unit/helper-specs.js

```diff
@@ -16,7 +17,7 @@ describe('helpers', () => {
         .returns(directories);
       mocks.fs.expects('lstat')
         .exactly(5)
-        .returns(Promise.resolve({isDirectory: () => {return true;}}));
+        .returns(B.resolve({isDirectory: () => {return true;}}));
```

Comment:
No, this was just how the folks writing this module did it. The async/await mechanics can handle a normal value.